src/Migrations/Version20171114112949.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * This migration will insert two records into joindinUsers, representing (up until the time of this migration) the two
+ * organizers of ZgPhp Meetup.
+ */
+class Version20171114112949 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration inserts two records representing ZgPhp Meetup organizers
+        $this->addSql('INSERT INTO joindinUsers (id, username, displayName, organizer) VALUES (18486, \'miro.svrtan\', \'Miro Svrtan\', TRUE), 
```

Comment:
No, we are already in DB, this would break on production.

Queries should be update

Reply:
Also, no personal info should be visible in code!

Reply:
For adding organizers, use fixtures and generic data for login/display name